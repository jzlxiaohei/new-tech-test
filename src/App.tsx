import React from 'react';
import { LayoutContainer, LayoutFooter, LayoutHeader } from './layout';
import { PageHome } from './pages/home';

function App() {
  return (
    <div className="App">
      <LayoutHeader />
      <LayoutContainer>
        <PageHome />
      </LayoutContainer>
      <LayoutFooter />
    </div>
  );
}

export default App;
