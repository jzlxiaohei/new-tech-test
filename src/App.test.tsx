import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

it('renders correctly', () => {
  const { baseElement } = render(<App></App>);
  expect(baseElement).toMatchSnapshot();
});
