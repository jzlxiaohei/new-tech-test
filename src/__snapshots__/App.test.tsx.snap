// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<body>
  <div>
    <div
      class="App"
    >
      <header
        class="layout-header css-h7smvq-LayoutHeader"
      >
        BROCCOLI & CO.
      </header>
      <div
        class="layout-container css-1xvaz39-LayoutContainer"
      >
        <div
          class="page-home css-ls5h3m-flexBox-flexBoxCenter-PageHome"
        >
          <div
            class="css-1wzvfrg-PageHome"
          >
            <div
              class="css-1yjrfxr-PageHome"
            >
              A better way 
              <br />
              to enjoy every day.
            </div>
            <div
              class="css-1uas052-PageHome"
            >
              Be the first to know when we launch.
            </div>
            <button
              class="aw-touchable css-1u7jufj-PageHome"
              style="user-select: none;"
            >
              Request an invite
            </button>
          </div>
        </div>
      </div>
      <footer
        class="layout-header css-18rlbq8-LayoutFooter"
      >
        <div
          class="css-1ffm2r3-LayoutFooter"
        >
          <div>
            Made with
            <span
              class="css-10n1kam-LayoutFooter"
            >
              ‚ù§
            </span>
            in Melbourne.
          </div>
          <div>
            @ 2021 Broccoli & Co. All rights reserved
          </div>
        </div>
      </footer>
    </div>
  </div>
</body>
`;
